app/views/browse/deployment-config.html

```diff
@@ -47,12 +47,12 @@
                       <a ng-href="project/{{projectName}}/set-limits?dcName={{deploymentConfig.metadata.name}}"
                         role="button">Set Resource Limits</a>
                     </li>
-                    <li ng-if="!autoscalers.length && ('extensions/horizontalpodautoscalers' | canI : 'create')">
+                    <li ng-if="!autoscalers.length && ('horizontalpodautoscalers' | canI : 'create' : 'autoscaling')">
```

Comment:
is it possible for this to be `({group:'autoscaling',resource:'horizontalpodautoscalers'} | canI : 'create')`?